src/main/java/org/tron/common/runtime/config/VMConfig.java

```diff
@@ -29,6 +32,8 @@
   private boolean vmTraceCompressed = false;
   private boolean vmTrace = false;
 
+  //Odyssey3.2 hard fork -- ForkBlockVersionConsts.ENERGY_LIMIT
+  public static boolean ENERGY_LIMIT_HARD_FORK = false;
```

Comment:
![Vulnerability](https://sonarsource.github.io/sonar-github/vulnerability.png 'Vulnerability') Vulnerability: Make ENERGY_LIMIT_HARD_FORK a static final constant or non-public and provide accessors if needed. ([squid:ClassVariableVisibilityCheck](https://sonarcloud.io/organizations/tron-zhaohong/rules#rule_key=squid%3AClassVariableVisibilityCheck))

[See it in SonarCloud](https://sonarcloud.io/project/issues?id=java-tron&pullRequest=1690&issues=AWbxOpgATJjFgJLTP8Jc&open=AWbxOpgATJjFgJLTP8Jc)

[ISSUE_KEY]: AWbxOpgATJjFgJLTP8Jc